### ISSUE TITLE
docker

### ISSUE THREAD

**Issue by daithi-coombes** [future-analytics/reusingdublin#107]
Create a dockerfile to ease setup for devOps.

**Comment by AlexTNolan** [comment]
Good morning,

So I've created and pushed an empty dockerfile to the repository and am now
installing the toolbox (hopefully that's along the right track). I'll send
you an e-mail when that's completed.

Alex

**Comment by daithi-coombes** [comment]
:+1:

**Comment by AlexTNolan** [comment]
Hi Daithi,

So the installation of docker is working fine now, no rush for yourself,
I'm going to read the documentation before proceeding as I'm still not
quite sure what it does, let alone starting a database on it.

Alex

**Comment by AlexTNolan** [comment]
So I've navigated to the folder in the docker console, created the
dockerfile and was wondering how to get the image of the reusing Dublin
database/connection I have found documentation on how to actually set up a
fresh database but there is very little in the way of importing one.

Alex

**Comment by daithi-coombes** [comment]
... also to note, the [docker file](https://github.com/future-analytics/reusingdublin/commit/8e6a74fce7eb2ec50cbcea4eb0f7d4cd733afbab) should have been added to its own branch, and then its own PR (different issues).

So in the end we would have two `PR`'s:
 1. admin panel
 2. docker setup

don't worry about back tracking, full steam ahead ;) When you have a docker file sorted then create a new branch (named `docker-setup` or something similar) and push that. I will then create a pr for it. Ignore the `dockerfile` in the `admin panel` for now, its existence there is no trouble

**Comment by daithi-coombes** [comment]
lol - great minds think alike, we both commented at the same time. For now, create the file, put what you think should go in it, push as a new branch and then I will create a default db on my end to add to the docker file.

**Comment by AlexTNolan** [comment]
Should I create a new directory for the new dockerfile ?

**Comment by daithi-coombes** [comment]
no, the root directory should be fine. I hate it but the current defacto is that all config files go in the project root (ie composer.json, package.json, docker etc).

**Comment by AlexTNolan** [comment]
ah, ok well I've created it in the root and pushed it, but I did create the
other directory, I don't seem to have permission to delete it
